README.md

```diff
@@ -18,15 +18,18 @@ There are multiple ways to run this server:
 ### Docker
 
 1.  Setup [Docker CE using the instructions for your OS](https://docs.docker.com/engine/installation/)
-2.  Build the Docker image, which will execute all the commands in the 
+2.  Build arxiv-base (`cd arxiv-base;  docker build -t arxiv-base:latest .`) if not using a registry.
+3.  Build the Docker image, which will execute all the commands in the 
```

Comment:
Record safety measure of `docker tag built_image_id arxiv-base:latest` since somehow docker images did not fully accept the local build of arxiv-base upon reproduction.